ci-operator/step-registry/openshift/release/analysis/test-case-analysis/openshift-release-analysis-test-case-analysis-commands.sh

```diff
@@ -4,6 +4,9 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+MINIMUM_SUCCESSFUL_COUNT=2
```

Comment:
This is ok for now. But over time we probably do not want a fix value for all variants.